dictionaries/CallMap.php

```diff
@@ -6923,22 +6923,22 @@
 'Locale::parseLocale' => ['array', 'locale'=>'string'],
 'Locale::setDefault' => ['bool', 'locale'=>'string'],
 'locale_accept_from_http' => ['string|false', 'header'=>'string'],
-'locale_canonicalize' => ['string', 'locale'=>'string'],
+'locale_canonicalize' => ['?string', 'locale'=>'string'],
 'locale_compose' => ['string|false', 'subtags'=>'array'],
-'locale_filter_matches' => ['bool', 'languageTag'=>'string', 'locale'=>'string', 'canonicalize='=>'bool'],
-'locale_get_all_variants' => ['array', 'locale'=>'string'],
+'locale_filter_matches' => ['?bool', 'languageTag'=>'string', 'locale'=>'string', 'canonicalize='=>'bool'],
+'locale_get_all_variants' => ['?array', 'locale'=>'string'],
 'locale_get_default' => ['string'],
 'locale_get_display_language' => ['string', 'locale'=>'string', 'displayLocale='=>'string'],
 'locale_get_display_name' => ['string', 'locale'=>'string', 'displayLocale='=>'string'],
 'locale_get_display_region' => ['string', 'locale'=>'string', 'displayLocale='=>'string'],
 'locale_get_display_script' => ['string', 'locale'=>'string', 'displayLocale='=>'string'],
 'locale_get_display_variant' => ['string', 'locale'=>'string', 'displayLocale='=>'string'],
-'locale_get_keywords' => ['array|false', 'locale'=>'string'],
-'locale_get_primary_language' => ['string', 'locale'=>'string'],
-'locale_get_region' => ['string', 'locale'=>'string'],
-'locale_get_script' => ['string', 'locale'=>'string'],
-'locale_lookup' => ['string', 'languageTag'=>'array', 'locale'=>'string', 'canonicalize='=>'bool', 'defaultLocale='=>'string'],
-'locale_parse' => ['array', 'locale'=>'string'],
+'locale_get_keywords' => ['array|false|null', 'locale'=>'string'],
+'locale_get_primary_language' => ['?string', 'locale'=>'string'],
+'locale_get_region' => ['?string', 'locale'=>'string'],
+'locale_get_script' => ['?string', 'locale'=>'string'],
+'locale_lookup' => ['?string', 'languageTag'=>'array', 'locale'=>'string', 'canonicalize='=>'bool', 'defaultLocale='=>'string'],
+'locale_parse' => ['?array', 'locale'=>'string'],
```

Comment:
These locale functions return null when locale string is too big. Do we want to ignore this possibility?